manifest/service.go

```diff
@@ -105,6 +105,14 @@ func (s *Service) Process(app string, m Manifest) Process {
 	return NewProcess(app, *s, m)
 }
 
+// UseSecureEnvironment - Determines if the service intends to use a secure environment
+func (s Service) UseSecureEnvironment() bool {
+	if useSecureEnvironment, ok := s.Labels["convox.environment.secure"]; useSecureEnvironment == "true" && ok {
```

Comment:
nit: could this function be simplified to `return s.Labels["convox.environment.secure"] == "true"`?